client/k8sclient/k8sclient.go

```diff
@@ -99,6 +108,9 @@ func New(config Config) (kubernetes.Interface, error) {
 				CertFile: config.TLS.CrtFile,
 				KeyFile:  config.TLS.KeyFile,
 				CAFile:   config.TLS.CAFile,
+				CertData: config.TLS.CrtData,
+				KeyData:  config.TLS.KeyData,
+				CAData:   config.TLS.CAData,
```

Comment:
Is there validation between the file path and content settings in the underlying Kubernetes libs?

Reply:
The byte arrays take precedence over the files as described in the GoDoc.  In the Kubernetes libs an error is returned if the files can't be read.

https://github.com/kubernetes/client-go/blob/72e1c2a1ef30b3f8da039e92d4a6a1f079f374e8/rest/config.go#L382-L394